teams.gd

```diff
@@ -1,6 +1,5 @@
 extends Panel
 
-var text: String = ""
```

Comment:
add a reference to the lineedit here with `@export var text: LineEdit`

Reply:
copy that